### ISSUE TITLE
0crat doesn't care about the developers' actual knowledge

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#544]
It is assumed by the model that developers will work on specialized repos (e.g. I, as a Java developer, won't work on a repo that has other stuff than Java). 

For instance, this repo is Java + advanced XML technologies. I have no idea about XML tech -- there are, of course, puzzles, I can puzzle stuff out, but this won't work indefinetely; at some point I'll not be able to puzzle the task out and since I have no other choice, I will have to spend a lot of time learning what is required. But do I want to do that? Is it worth my time since I am [not paid for it](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html)? I'd bet a lot of money that I will never use it in other project. 

I see the need for some kind of labeling mechanism; 0crat should know which task is frontend, which is backend, which is DB modeling, which is XML and so on. Then, of course, the developers would have to give more information about their expertise, somewhere on 0crat.com.

Bottom line is: it is not a school but a project, a repo, should be able to decide what it will force its developers to learn and what not. So far, the only way to be sure that the devs receive tasks of their knowledge is to split the repos based on technology, which may not always be desirable.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@amihaiemil that's a valid concern, but my answer to this is that each repo should be one technology focused only. There should not be repositories with multiple technologies in it. If that is the case, the repo is wrong, not Zerocrat. Labeling tasks and letting Zerocrat make those decisions will only complicate the situation and make it messy. So, the answer is no.

**Comment by 0crat** [comment]
The job [#544](https://github.com/zerocracy/farm/issues/544) assigned to @yegor256[/z](https://www.0crat.com/u/yegor256), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/431410087). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+495](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://datum.zerocracy.com/pages/policy.html#19): -5 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9660](https://www.0crat.com/u/yegor256)

**Comment by amihaiemil** [comment]
@yegor256 I see. Did you really want to put this in scope and pay me, or not (since you rejected the issue)? Should I refund the transaction?

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9690](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@amihaiemil it was done intentionally. The report was valid, we paid you for it. But the decision on it was "rejection." The decision and the work you've done to report the bug -- are two separate things. We don't pay for bugs that are minor, cosmetic or incorrectly reported. This one is not one of those.

**Comment by amihaiemil** [comment]
@yegor256 Ok, thanks :)

**Comment by 0crat** [comment]
The job [#544](https://github.com/zerocracy/farm/issues/544) is now out of scope
